packages/client/utils/constants.ts

```diff
@@ -40,9 +40,14 @@ export const meetingColumnArray = [DONE, ACTIVE, STUCK, FUTURE] as TaskStatusEnu
 /* Scoping Task Search Filter */
 export const taskScopingStatusFilters = [ACTIVE, STUCK, FUTURE] as TaskStatusEnum[]
 
-/* Drag-n-Drop Items */
-export const TASK = 'task'
-export const AGENDA_ITEM = 'agendaItem'
+/* Drag-n-Drop Items DroppableId */
+export const AGENDA_ITEM = 'AGENDA_ITEM'
+export const DISCUSSION_TOPIC = 'DISCUSSION_TOPIC'
+export const ESTIMATING_TASK = 'ESTIMATING_TASK'
+export const TEMPLATE_DIMENSION = 'TEMPLATE_DIMENSION'
+export const TEMPLATE_PROMPT = 'TEMPLATE_PROMPT'
+export const TEMPLATE_SCALE_VALUE = 'TEMPLATE_SCALE_VALUE'
+export const ACTIVE_TASK = 'ACTIVE_TASK'
```

Comment:
I could have create an enum to encapsulate them together but decided to not further increase the scope of these changes.